deploy-testnetv2.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+export STELLAR_NETWORK_PASSPHRASE="Test SDF Network ; September 2015"
+export STELLAR_RPC_URL="https://soroban-testnet.stellar.org"
+export STELLAR_ACCOUNT=equitxtestnet
+
+rm -rf target/loam
+loam build 
+
+# Deploy and initialize contracts for each asset
+DATAFEED="CCYOZJCOPG34LLQQ7N24YXBM7LL62R7ONMZ3G6WZAAYPB5OYKOMJRN63"
```

Comment:
Where is this deployed? Could the orchestrator also deploy this contract?

Reply:
No, this is a reflector contract, we don't deploy it 

https://reflector.network/